_glossary/VIRTUAL_DOM.md

```diff
@@ -1,10 +1,37 @@
 ---
 title: Virtual DOM
-excerpt: a copy of the DOM in memory that the program can modify instead of directly interfacing with the “real” DOM to help speed up interactions
+excerpt: a pattern in which rather than changing the DOM directly, you create a javascript representation of the DOM you want and let the framework sort out details of how to make it happen
```

Comment:
I would like to avoid using *you* in descriptions. How about:

> a pattern in which a representation of the DOM gets used in place of the actual one, leaving interpretation details to the application layer